tests/backup_helper.go

```diff
@@ -4920,6 +4948,41 @@ func CreateRuleForBackupWithMultipleApplications(orgID string, appList []string,
 	return preRuleName, postRuleName, nil
 }
 
+type VMRuleType string
+
+const (
+	Freeze   VMRuleType = "freeze"
+	Unfreeze VMRuleType = "unfreeze"
+)
+
+func CreateVMRule(ruleName string, vms []kubevirtv1.VirtualMachine, ruleType VMRuleType, ctx context1.Context) error {
```

Comment:
Could we use CreateRuleForKubevirtBackup from backup for this?

Reply:
I think we can call it `CreateRuleForVMBackup` instead of `Kubevirt` just be consistent. Is it OK?

Reply:
Done